docs/01-app/02-building-your-application/10-deploying/index.mdx

```diff
@@ -297,6 +297,23 @@ If you don't need a mix of both static and dynamic components, you can make your
 
 </AppOnly>
 
+<AppOnly>
+
+### Using `after`
+
+#### Self-hosting
+
+`after` is fully supported in `next start`.
+
+Note that when you want to stop the server, you should give it a chance to gracefully shutdown by sending `SIGINT`/`SIGTERM` and waiting.
+This will allow the server to wait for any pending callbacks/promises passed to `after` to finish.
```

Comment:
```suggestion
  Note that when you want to stop the server, you should allow it to gracefully shutdown by sending `SIGINT`/`SIGTERM` and waiting.This will allow the server to wait for any pending callbacks/promises passed to `after` to finish.
```